nifi-docs/src/main/asciidoc/user-guide.adoc

```diff
@@ -986,13 +991,11 @@ automatically be 'cloned', and a copy will be sent to each of those Connections.
 
 ==== Settings
 
-The Settings Tab provides the ability to configure the Connection's name, FlowFile expiration, Back Pressure thresholds, and
-Prioritization:
+The Settings Tab provides the ability to configure the Connection's Name, FlowFile Expiration, Back Pressure Thresholds, Load Balance Strategy and Prioritization:
```

Comment:
```suggestion
The Settings tab provides the ability to configure the Connection's Name, FlowFile Expiration, Back Pressure Thresholds, Load Balance Strategy and Prioritization:
```

Reply:
There are other instances in the doc where tab is capitalized similarly:

-Line 641 "Scheduling Tab"
-Line 680 "Properties Tab"
-Line 681 "Properties Tab"
-Line 981 "Details Tab"

But other instances where it is not capitalized:

-Lines 716 & 717 "Comments tab"

I'm OK with either style, but we should be consistent.  So if you like the non-capitalized versions, we need to edit the 4 other instances I noted above.

Reply:
I would prefer tab being lowercased, since it's not part of the title of the actual tab.

Reply:
Sounds good